File: Fire Drill/taskfive.py


counter = 1
sum_even = 0.0;

while (counter <= 10):

	score = float(input("Enter score:"))

	if (score % 2 == 0):
		sum_even += score
	counter += 1


print("The sum of the even scores of the 10 scores is", sum_even)

File: Previous Works/Fire Drill/TaskFive.java
import java.util.Scanner;

public class TaskFive{

	public static void main (String [] args){

		Scanner input = new Scanner (System.in);

		double sumEven = 0.0;
		double sum = 0.0;
		

		for (int counter = 1; counter <= 10; ++counter){

			System.out.printf("Enter score %d:%n", counter);

			double score = input.nextDouble();
			
			if (score % 2 == 0){
				sumEven += score;
			}
			

			sum += score;

		}

		System.out.printf("The sum of the even scores is %.2f%n", sumEven);
			

	}



}